### ISSUE TITLE
Controls present in the "Custom accessible properties form" page are not accessible using keyboard: A11y_.NET CoreWinforms_Custom accessible properties form_Keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11444]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev1; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG2.1.1; #DesktopApp; #Win11; #FTP; #Keyboard;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Accessible properties: Testings controls with custom accessible name, role and description" and hit ENTER key.
4. "Custom accessible properties form" screen would appear.
5. Navigate using TAB key and observe the issue.

### Actual Result:
Controls present in the "Custom accessible properties form" page are not accessible using keyboard keys. Only 4 controls are accessible which are button 1, checkbox 1, dropdown and calendar control other than these control no control is accessible.

### Expected Result:
All controls present in the "Custom accessible properties form" page should be accessible using keyboard keys.

### User Impact:
Keyboard users will not be able to interact with the control and will not understand what functionality is provided for that controls and will not get information about it.

### Attachment

https://github.com/dotnet/winforms/assets/90901519/dbc407bf-856f-42ad-9639-4b6f1599c168

**Comment by MandiMan** [comment]
We can reset the Tab Order to ensure top-to-bottom navigation using the Tab key. However, some controls can't be navigated directly with the Tab key and require special properties to be set, such as HScrollBar, VScrollBar which requires the TabStop property to be set to True, ListView, treeView which requires columns or Node to be added. FlowLayoutPanel, tableLayoutPanel need to add other controls (e.g. Button) inside.

https://github.com/dotnet/winforms/assets/108860782/d719c14b-416d-4e91-b209-36eb67f0232e

**Comment by merriemcgaw** [comment]
@MandiMan do you think we can/should also break this into two forms so it's easier to access. Depending on what we're trying to demonstrate here, perhaps we should go ahead and fill it with sample data so they can see it's accessible.

@Tanya-Solyanik do you recall what we were trying to demonstrate here?

**Comment by MandiMan** [comment]
> @MandiMan do you think we can/should also break this into two forms so it's easier to access. Depending on what we're trying to demonstrate here, perhaps we should go ahead and fill it with sample data so they can see it's accessible.
> 
> @Tanya-Solyanik do you recall what we were trying to demonstrate here?

I think it would be possible to break the default directly accessible controls and the non-directly accessible controls into two forms if necessary. It might look more intuitive that way.

**Comment by Yash14j** [comment]
Github Tags:#Rev:yaja;

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw, @MandiMan  - This form is for testing individual controls to make sure we can set custom Name/Role/Description accessible properties. We already have these controls containing sample data in other forms. I would simply exclude this page from the test pass.

**Comment by Olina-Zhang** [comment]
@merriemcgaw  added some text for this Form's text property with "excluded this dialog in Accessibility Test Pass"
![image](https://github.com/dotnet/winforms/assets/26474449/14f71a8a-be02-4f5c-a224-fe1989da3e18)

**Comment by Olina-Zhang** [comment]
@PoojaNamde Test app is uploaded, please check.

**Comment by PoojaNamde** [comment]
@Olina-Zhang We have logged the bug for "Custom accessible properties form" screen, the above provided screen shot is for "Data binding example screen". Could you please confirm whether which screen we have to exclude from testing. 
Also please find the below screen for "Custom accessible properties form" screen where we are not able to see any message for excluding from testing.

![image](https://github.com/dotnet/winforms/assets/90901519/d9858623-1fcc-4ec6-821e-6eb99300d706)

**Comment by Olina-Zhang** [comment]
@PoojaNamde did you download the latest updated app for testing?

**Comment by PoojaNamde** [comment]
@Olina-Zhang Yes, I have downloaded the latest app.

**Comment by Olina-Zhang** [comment]
@PoojaNamde Sorry, I give your another form's updated result. Now I have do the same thing for this form "Custom accessible properties form" again, please check updated app again. Thanks!
![image](https://github.com/dotnet/winforms/assets/26474449/4df5709f-9542-4f3c-8ea0-50b1e3ba7e7a)

**Comment by Olina-Zhang** [comment]
@PoojaNamde finally we decided to remove this "Custom accessible properties form" as discussion in GH issue: https://github.com/dotnet/winforms/issues/11458, I have updated and uploaded app. Please verify and close this bug.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-13-24;

As per the above comment and text provided for sample form "excluded this Form in Accessibility Test Pass", closing this bug.
